clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -461,6 +932,30 @@ Expected<StringRef> linkDevice(ArrayRef<StringRef> InputFiles,
   case Triple::ppc64:
   case Triple::ppc64le:
     return generic::clang(InputFiles, Args);
+  case Triple::spirv32:
+  case Triple::spirv64:
+  case Triple::spir:
+  case Triple::spir64: {
+    if (IsSYCLKind) {
+      SmallVector<StringRef, 16> InputFilesVec;
+      for (StringRef InputFile : InputFiles)
+        InputFilesVec.push_back(InputFile);
+      auto SYCLPostLinkFile = sycl::runSYCLPostLink(InputFilesVec, Args);
```

Comment:
These lines could be as simple as one line if `runSYCLPostLink` accepted `ArrayRef` as an argument

```suggestion
      auto SYCLPostLinkFile = sycl::runSYCLPostLink(InputFiles, Args);
```

Reply:
Added this change to next commit.